.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - actions
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
We can try seeing what happens on a PR when the upstream repo has just the push trigger. The follow up PR can add a `--version` test on a "runtime-Qt-only" machine (and restore this block if needed).